robot.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import wpilib
+import wpilib.event
+import magicbot
+
+from components.chassis import Chassis
+from controllers.movement import Movement
+
+from utilities.scalers import rescale_js
+
+
+class MyRobot(magicbot.MagicRobot):
+    # Components
+    chassis: Chassis
+
+    movement: Movement
```

Comment:
Controllers need to run before the components they control.
```suggestion
    # Controllers
    movement: Movement

    # Components
    chassis: Chassis
```